zzzz_modular_occulus/code/modules/reagents/reagents/other.dm

```diff
@@ -5,4 +5,387 @@
+	id = "bliss"
+	name = "Bliss"
+	description = "Looks as though it would metabolize into the ultimate hallucinogenic cocktail"
+	color = "#ffc0eb"
+	metabolism = 10 * REM
+	var/init = 0
+	overdose = 15
+	addiction_chance = 60
+
+/datum/reagent/bliss/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	if(!init)
+		to_chat(M, "<span class='warning'>Everything seems to turn glossy and wonderful.</span>")
+		init = 1
+	if(prob(5))
+		if(prob(98))
+			var/list/bliss_messages = list("You feel as if everything is great",
+			"The world seems bright and sparkly", "Everyone enjoys being with you.",
+			"You feel like theres nothing you can't do", "You feel a weight lifting off your chest",
+			"Nothing can go wrong for you",
+			"All your troubles seem to wash away")
+
+			to_chat(M, "<font color='#e3209b'>[pick(bliss_messages)]</font>")
+	sanity_gain = 4
+	M.druggy = max(M.druggy, 5 * effect_multiplier)
+	M.make_dizzy(10 * effect_multiplier)
+	return ..()
+
+/datum/reagent/frioline
+	id = "frioline"
+	name = "Frioline"
+	description = "Could cause rapid and sustained decrease in body temperature"
+	color = "#A0E1F7"
+
+/datum/reagent/frioline/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	if(M.bodytemperature > 310)
+		to_chat(M, "<span class='notice'>You suddenly feel very cold.</span>")
+	M.bodytemperature = max(165, M.bodytemperature - 30)
+	return ..()
+
+/datum/reagent/luxitol
+	id = "luxitol"
+	name = "Luxitol"
+	description = "Mimics compounds in known connection with bioluminescence"
+	color = "#61E34F"
+	metabolism = 0.2 * REM
+
+/datum/reagent/luxitol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.set_light(10)
+	return ..()
+
+/datum/reagent/luxitol/on_remove(var/atom/A)
+	if(istype(A, /mob))
+		var/mob/M = A
+		M.set_light(0)
+	return ..()
+
+/datum/reagent/liquid_skin
+	id = "liquid_skin"
+	name = "Liquid Skin"
+	description = "Fills in microscopic ridges on biotic surfaces and hardens"
+	color = "#F7E9BE"
+
+/datum/reagent/liquid_skin/touch_mob(var/mob/M, var/volume)
+	if(istype(M, /mob/living))
+		var/mob/living/L = M
+		var/burned = L.getFireLoss() > 0
+		if(burned)
+			L << "<span class='notice'>In a strange sensation, you feel some burns stop hurting.</span>"
+			L.heal_organ_damage(0, min(15, volume / 4))
+
+		if (mFingerprints in L.mutations)
+			if(!burned)
+				L << "<span class='warning'>Another application of the substance does nothing weird to your hands.</span>"
+		else
+			L.mutations.Add(mFingerprints)
+			L << "<span class='notice'>Your fingers feel strange after the substance splashes on your hands.</span>"
+	return ..()
+
+/datum/reagent/paralitol
+	id = "paralitol"
+	name = "Paralitol"
+	description = "Seems as if it could work as an extreme muscle inhibitor"
+	color = "#2F85CC"
+	metabolism = 3 * REM
+
+/datum/reagent/paralitol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.Weaken(2)
+	M.stuttering = 50
+	return ..()
+
+/datum/reagent/mortemol
```

Comment:
This steps on the toes of the Resussicator

Reply:
It does a bit. Shadz reccomended I make it require Kaiser chems, so it can be used if we have no CMO, and have had the fortune of killing a kaiser.